shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -138,8 +138,8 @@ public int run(String... argv) {
     try {
       command.run(args);
       return 0;
-    } catch (IOException ioe) {
-      System.out.println(ioe.getMessage());
+    } catch (IOException e) {
+      System.out.println(e.getMessage());
```

Comment:
On further thought, I think it makes more sense to go with the debug flag since often the error message will be enough for users to figure out their problem. I've created https://tachyon.atlassian.net/browse/TACHYON-1608